docs/examples/quickstart/python/server.py

```diff
@@ -4,7 +4,10 @@
 from expense import Expense, EXPENSES
 
 oso = Oso()
-oso.load_file("expenses.polar")
+
+
+def setup_oso():
```

Comment:
Why do we need this change?

Reply:
I did this so I can load the file in the test (to make sure it's valid python) without re-initing oso.

Reply:
I'm a -1 on this change since users can download this file via a link on the quickstart page. I think I would rather test the quickstart examples by actually running the server and programmatically making requests against it.

Reply:
If the users download and run the file it'll work the same as it did before. I agree that actually testing the web servers is better but this still validates it more than we did before.